### ISSUE TITLE
The attempted blocking write is not allowed because the non-blocking I/O has already been started by the application registering the WriteListener 

### ISSUE THREAD

**Issue by xring** [play2war/play2-war-plugin#304]
I'm using Play2.4 with Scala2.11 and play2war on IBM Bluemix with Liberty.

It seems that IBM updated their Bluemix platform on 9 Feb 2016, then my app do not work anymore.

I search the logs, there only one error message:

> [ERROR ] SRVE0918E: The attempted blocking write is not allowed because the non-blocking I/O has already been started by the application registering the WriteListener [play.core.server.servlet31.Play2Servlet31RequestHandler$ResultWriteListener@15bfa0fc].

I want to know this happened due to my codes or due to the play2-war-plugin?

**Comment by yanns** [comment]
Just a guess: it seems that they are more than one application deployed in the container.
One application is the play app, using async IO, and the other app is using blocking IO.

**Comment by yespeebala** [comment]
I'm also facing same issue. It is working in WAS Liberty profile 8.5.5.7. But,  When I deployed in WAS Liberty profile 8.5.5.9 or upgraded version. got same error like below.
[ERROR ] "SRVE0918E: The attempted blocking write is not allowed because the non-blocking I/O has already been started by the application registering the WriteListener [play.core.server.servlet31.Play2Servlet31RequestHandler$ResultWriteListener@38beb71d].”


@xring Have you found solution?

**Comment by xring** [comment]
@yespeebala No, I still can not run Play2.4(or above) on Bluemix(Liberty based runtime). I used Akka-HTTP instead.
